plugins/inputs/ghwebhooks/ghwebhooks.go

```diff
@@ -0,0 +1,338 @@
+package ghwebhooks
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"sync"
+
+	"github.com/gorilla/mux"
```

Comment:
this is a new dependency for Telegraf, you'll need to add it like this: https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md#adding-a-dependency